sui/README.md

```diff
@@ -150,8 +150,8 @@ node sui/gas-service.js payGas --amount 0.1 ethereum 0x6f24A47Fc8AE5441Eb47EFfC3
 Collect gas:
 
 Conditions:
-- The `GasCollectorCap` must be stored in the operator contract.
-- The sender must holds the `Operator` object.
+- The `GasCollectorCap` must be stored in the gas service object config.
```

Comment:
```suggestion
- The `GasCollectorCap` object id is read from the chain config, under gas service objects.
- The `GasCollectorCap` object must have been stored in the Operators contract via `store_cap` by the previous owner.
```
since that's what you were going for before